namelayer-spigot/src/main/resources/plugin.yml

```diff
@@ -1,6 +1,6 @@
 name: NameLayer
 main: vg.civcraft.mc.namelayer.NameLayerPlugin
-version: ${project.version}
+version: ${releaseVersion}
```

Comment:
So, difference is that I have a property on the parent pom that defines releaseVersion as the actual release version. And that's what gets injected here.